packages/shared/src/components/tooltips/InteractivePopup.tsx

```diff
@@ -50,7 +50,7 @@ function InteractivePopup({
     <Portal>
       <div
         className={classNames(
-          'fixed z-3 bg-theme-bg-primary rounded-16 overflow-hidden',
+          'fixed z-modal bg-theme-bg-primary rounded-16 overflow-hidden',
```

Comment:
We need to tripple check this doesn't affect any other use-cases please 🙏

Reply:
Agreed. Will further check this one.

Reply:
Introduced a new one to be less than the modal to ensure no collision will happen.